src/patternfly/components/Check/check-input.hbs

```diff
@@ -0,0 +1,6 @@
+<input class="pf-c-check__input{{#if check-input--modifier}}{{check-input--modifier}}{{/if}}"
```

Comment:
You're missing a space, should be `{{#if check-input--modifier}} {{check-input--modifier}}`